### PR TITLE
[libpqxx] linux support

### PR DESCRIPTION
**Pull request by alexander-smyslov**
**Describe the pull request**

- What does your PR fix? Fixes #
add linux support 
- Which triplets are supported/not supported? 
x64-linux 
Have you updated the CI baseline?

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?

### CODE REVIEW

**Review by NancyLi1013 — COMMENTED** [review]
Could you please also bump the version as  `6.4.5-3` in CONTROL file?

**Review by NancyLi1013 — APPROVED** [review]

File: ports/libpqxx/CMakeLists.txt [hunk 1]
```diff
@@ -34,7 +34,9 @@ file(GLOB SRCS "${PROJECT_SOURCE_DIR}/src/*.cxx")
 if(BUILD_SHARED_LIBS)
     set(TARGET libpqxx)
     set(SHARED_DEFINITION -DPQXX_SHARED)
+if(MSVC)
```
Comment (strega-nil) [inline ports/libpqxx/CMakeLists.txt:37]: Could you correct the indentation here?

File: ports/libpqxx/CMakeLists.txt [hunk 2]
```diff
@@ -34,7 +34,9 @@ file(GLOB SRCS "${PROJECT_SOURCE_DIR}/src/*.cxx")
...
     list(APPEND SRCS "${PROJECT_SOURCE_DIR}/win32/libpqxx.cxx")
+endif(MSVC)
```
Comment (strega-nil) [inline ports/libpqxx/CMakeLists.txt:39]: and make this `endif()`?

**Review by strega-nil — APPROVED** [review]
Just some minor changes, then we should get this merged :)

What's up with the baseline, though? Shouldn't it need to be changed? Was it just silently failing before?

**Comment by NancyLi1013** [comment]
@strega-nil 
I have checked baseline and it seems that there are no any triplets set for `libpqxx` before.
So we don't neede to update the baseline.

File: ports/libpqxx/CMakeLists.txt [hunk 3]
```diff
@@ -34,7 +34,9 @@ file(GLOB SRCS "${PROJECT_SOURCE_DIR}/src/*.cxx")
 if(BUILD_SHARED_LIBS)
     set(TARGET libpqxx)
     set(SHARED_DEFINITION -DPQXX_SHARED)
-    list(APPEND SRCS "${PROJECT_SOURCE_DIR}/win32/libpqxx.cxx")
+    if(MSVC)
+        list(APPEND SRCS "${PROJECT_SOURCE_DIR}/win32/libpqxx.cxx")
+    endif(MSVC)
```
Comment (NancyLi1013) [inline ports/libpqxx/CMakeLists.txt:39]:
```suggestion
    endif()
```

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Comment by strega-nil** [comment]
Alright cool, thanks much @alexander-smyslov :)

**Pull request merged**